### ISSUE TITLE
Threshold 

### ISSUE THREAD

**Issue by JuseokSeong** [CVI-SZU/UniTSFace#5]
Thank you for the nice work!!

Is it possible to make predictions based on the threshold obtained from model training (self.bias in head.py)? Without conventionally determining the threshold.

For example,
score_value > self.bias / self.s -> positive trial
score_value < self.bias / self.s -> negative trial
Is this possible? 

However, it doesn't show good results..

**Comment by xi-jia** [comment]
Yes, the threshold in the loss function is mainly used to learn discriminative features. The testing threshold should be determined by specific testing scenarios/metrics.
